src/Microsoft.Health.SqlServer/Features/Schema/Manager/SchemaManagerDataStore.cs

```diff
@@ -124,8 +119,16 @@ public async Task ExecuteScriptAsync(string script, CancellationToken cancellati
         using SqlConnectionWrapper sqlConnectionWrapper = await _sqlConnectionWrapperFactory.ObtainSqlConnectionWrapperAsync(cancellationToken: cancellationToken).ConfigureAwait(false);
         using SqlCommandWrapper sqlCommandWrapper = sqlConnectionWrapper.CreateRetrySqlCommand();
 
-        var server = new Server(GetServerConnectionWithTimeout(sqlCommandWrapper.Connection));
-        server.ConnectionContext.ExecuteNonQuery(script);
+        await ExecuteWithGoSupport(script, sqlCommandWrapper, cancellationToken).ConfigureAwait(false);
+    }
+
+    private static async Task ExecuteWithGoSupport(string script, SqlCommandWrapper sqlCommandWrapper, CancellationToken cancellationToken)
+    {
+        foreach (string statement in script.Split(["\nGO"], StringSplitOptions.RemoveEmptyEntries))
+        {
+            sqlCommandWrapper.CommandText = statement;
+            await sqlCommandWrapper.ExecuteNonQueryAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
should we update the timeout here i.e. sqlCommandWrapper.CommandTimeout = (int)_sqlServerDataStoreConfiguration.StatementTimeout.TotalSeconds;

if I am understanding it right, default timeout will be set to [commandTimeout ](https://github.com/microsoft/healthcare-shared-components/blob/2168b5b68e58b6c64dc8f49c8e9734950ac9eae0/src/Microsoft.Health.SqlServer/Configs/SqlServerDataStoreConfiguration.cs#L67) which will not be enough for script execution

Reply:
I did look for this and it looks like it's being set in the wrapper based on our config

https://github.com/microsoft/healthcare-shared-components/blob/2168b5b68e58b6c64dc8f49c8e9734950ac9eae0/src/Microsoft.Health.SqlServer/Features/Client/SqlConnectionWrapper.cs#L99-L105

Reply:
Ah ok, got it